backend/src/database/repositories/memberRepository.ts

```diff
@@ -785,6 +880,24 @@ class MemberRepository {
       transaction,
     })
 
+    output.lastActive = output.activities[0]?.timestamp ?? null
```

Comment:
I like this lastActive. However, the frontend needs also the lastActivity (timestamp, platform, type) ([Notion](https://www.notion.so/crowddev/Design-and-front-end-for-API-V1-62cc346e6677405ba33ec0e7f9b9b056#93430b2e9ea34324bae695b7a729d764)). I think we could do both.

Reply:
added whole last activity object as well.. But we're lazy loading this activity along with organization and tasks as well. We'll need to do some performance tests on it, the overhead of additional queries may slow down the response time when getting a lot of members from member.list